_pages/events/learnathon-2017.md

```diff
@@ -119,7 +119,7 @@ Git repos to clone:
     -   [`https://github.com/bigdataviewer/bigdataviewer-vistools.git`](https://github.com/bigdataviewer/bigdataviewer-vistools.git)
 -   Afternoon Ops Session:
     -   [`https://github.com/imagej/tutorials`](https://github.com/imagej/tutorials)
-        -   Import the `maven-projects/using-ops` into your IDE
+        -   Import the `1-Using-ImageJ/2-ImageJ-Ops.ipynb` into your IDE
```

Comment:
This change won't work, because importing a Jupyter notebook into an IDE does not make sense—at least, not in the way it was meant here, where Java code is imported into a Java IDE.